core/src/main/java/com/amplifyframework/predictions/models/FaceLivenessSessionInformation.kt

```diff
@@ -20,6 +20,23 @@ import com.amplifyframework.annotations.InternalAmplifyApi
 data class FaceLivenessSessionInformation(
     val videoWidth: Float,
     val videoHeight: Float,
-    val challengeVersions: String,
-    val region: String
+    val challenge: String? = null,
+    val region: String,
+    val challengeVersions: List<Challenge>? = null,
+    val preCheckViewEnabled: Boolean? = null,
+    val attemptCount: Int? = null
 )
+
+@InternalAmplifyApi
+data class Challenge(
+    val type: ChallengeType,
+    val version: String
+) {
+    fun toQueryParamString(): String = "${type.name}_$version"
+}
+
+@InternalAmplifyApi
+enum class ChallengeType {
```

Comment:
Thoughts?

```
@InternalAmplifyApi
sealed class Challenge private constructor(val name: String, val version: String) {
    
    @InternalAmplifyApi
    class FaceMovementChallenge(version: String) : Challenge("FaceMovementChallenge", version)
    
    @InternalAmplifyApi
    class FaceMovementAndLightChallenge(version: String) : Challenge("FaceMovementAndLightChallenge", version)

    fun toQueryParamString(): String = "${name}_$version"
}
```

This may be safer if a single challenge type needs more information than another challenge type.